### ISSUE TITLE
Error calling pander on empty data.frame

### ISSUE THREAD

**Issue by BrianDiggs** [Rapporter/pander#203]
# Summary

When `pander` is called with an empty `data.frame`, an error is thrown.

# Reproducible Example

```r
pander(data.frame())
```

# Expected Result

```r
> pander(data.frame())
> 
```

# Actual Result

```r
> pander(data.frame())
Error in which(sapply(t, is.numeric)) : 
  argument to 'which' is not logical
```

## Traceback after error

```r
> traceback()
6: which(sapply(t, is.numeric))
5: pandoc.table.return(...)
4: cat(pandoc.table.return(...))
3: pandoc.table(x, caption = caption, ...)
2: pander.data.frame(data.frame())
1: pander(data.frame())
```

# SessionInfo

```
R version 3.2.1 (2015-06-18)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 7 x64 (build 7601) Service Pack 1

locale:
[1] LC_COLLATE=English_United States.1252 
[2] LC_CTYPE=English_United States.1252   
[3] LC_MONETARY=English_United States.1252
[4] LC_NUMERIC=C                          
[5] LC_TIME=English_United States.1252    

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] pander_0.5.2

loaded via a namespace (and not attached):
[1] tools_3.2.1  Rcpp_0.11.6  digest_0.6.8
```

**Comment by daroczig** [comment]
Thanks for this report.

@RomanTsegelskyi, I thought it was resolved with 678c21bdd8f48e68a18620f66b7b718175f4b48b, can you please have a look?

**Comment by BrianDiggs** [comment]
That commit does not seem to be part of pander 0.5.2

**Comment by daroczig** [comment]
@BrianDiggs yes, it's a newer commit -- but the most recent dev version of `pander` still has this bug, it seems -- despite the fact we tried to resolve this previously. I am sure @RomanTsegelskyi will have a solution for this in the near future, but if not, I will be happy to handle it as well.

**Comment by RomanTsegelskyi** [comment]
Fixed in #206. Problem was that for empty `data.frame`, `colnames` return vector of length 0 instead of `NULL` as it does for `matrix`.

**Comment by daroczig** [comment]
Let's merge if tested.

**Comment by RomanTsegelskyi** [comment]
Merged #206
